src/get-auth-urls.ts

```diff
@@ -7,6 +7,8 @@ import {PaymentActorType} from "./schema/payment"
 import {StandingOrderFrequency} from "./schema/standing-order"
 import {RequestPayee} from "./schema/payee"
 
+type PermissionsAction = "add" | "replace"
```

Comment:
Can we get this type in one place and reuse? Its here but also: src/requests/types/auth-requests.ts:33

So we can just import/export and have it all in one place if we want to add more :+1:

Reply:
Yeah thanks for spotting this, I wasn't really sure where a good place for it was when i looked the first time. I'll have a second look

Reply:
Yeah i think thats the best file :+1: